public/goalCreation.html

```diff
@@ -0,0 +1,36 @@
+<<<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Finance App</title>
+</head>
+<body>
+    <main>
+        <h1>Finance App</h1>
+        <p>Welcome to the Goal Page! 🏖️</p>
+        <section>
+            <form id="goal_form">
+                <h3>Goal Name</h3>
+                <input type="text" name="goal_name" id="goal_name" placeholder="Goal Name" maxlength="10" required><br><br>
+
+                <h3>Goal Description</h3>
+                <textarea name="goal_description" id="goal_description" placeholder="Goal Description" maxlength="50"></textarea><br><br>
+
+                <h3>Goal Amount</h3>
+                <input type="text" name="goal_amount" id="goal_amount" placeholder="$0.00" pattern="\$\d+(\.\d{2})?" required><br><br>
```

Comment:
I see that here you are using a pattern for the user input `\$\d+(\.\d{2})?`

However I believe instead of using that pattern, you are supposed to use Javascript to change the user input goal amount INTO `\$\d+(\.\d{2})?`

Reply:
Thank you Dinaol for actually pointing  this out to me I will definitely fix this issue.